corgi/core/constants.py

```diff
@@ -26,3 +26,8 @@
 MODULE_CONDITION = Q(type="RPMMOD")
 INDEX_CONTAINER_CONDITION = Q(type="OCI", arch="noarch")
 ROOT_COMPONENTS_CONDITION = SRPM_CONDITION | MODULE_CONDITION | INDEX_CONTAINER_CONDITION
+
+# The ratio of build types to total in the relation table
+CDN_RELATIONS_RATIO = 0.8
+BREW_RELATIONS_RATIO = 0.15
+YUM_RELATIONS_RATIO = 0.05
```

Comment:
Do we think these ratios might change a lot once we load more data? And maybe we should add a test that these add up to 1.0, in case one is changed in the future without changing the others, or a new type gets added.

Reply:
I don't think these will change much. I took a guess at these before calculating that actual ratios and my guess was pretty accurate. That indicates to me they have probably been in this ratio for a long time.